packages/node/src/behavior/context/server/RemoteActorContext.ts

```diff
@@ -163,13 +179,13 @@ export function OnlineContext(options: OnlineContext.Options) {
             };
             exchange.closing.on(notifyInteractionComplete);
         }
-        const context: ActionContext = {
+        const context: RemoteActorContext & T = {
             ...options,
             session,
             exchange,
             subject,
 
-            fabric,
+            fabric: fabric?.fabricIndex ?? FabricIndex.NO_FABRIC,
```

Comment:
Did you checked all cases where maybe foo.fabric is checked for undefined?

Reply:
I tried to, yes